src/gt4py/backend/module_generator.py

```diff
@@ -465,11 +465,15 @@ def _is_not_empty(self) -> bool:
     def generate_imports(self) -> str:
         source = ["from gt4py import utils as gt_utils"]
         if self._is_not_empty():
+            assert self.pyext_file_path is not None
+            file_path = 'f"{{pathlib.Path(__file__).parent.resolve()}}/{}"'.format(
+                os.path.basename(self.pyext_file_path)
+            )
```

Comment:
Mixing escaped and non-escaped placeholders is a bit confusing. I propose to replace the `format()`  call by a simple string concatenation:
```suggestion
            file_path = 'f"{{pathlib.Path(__file__).parent.resolve()}}/"' + str(
                os.path.basename(self.pyext_file_path)
            )
```